src/library/Uno.Material/Styles/Controls/v2/Button.xaml

```diff
@@ -0,0 +1,798 @@
+
+						<Grid x:Name="Root"
+							  Background="{TemplateBinding Background}"
+							  BorderBrush="{TemplateBinding BorderBrush}"
+							  BorderThickness="{TemplateBinding BorderThickness}"
+							  CornerRadius="{TemplateBinding CornerRadius}">
+
+							<Border x:Name="StateLayer"
+									Background="Transparent"
+									CornerRadius="{TemplateBinding CornerRadius}" />
+
+							<um:Ripple x:Name="Ripple"
+									   Feedback="{TemplateBinding Foreground}"
+									   FeedbackOpacity="{StaticResource PressedOpacity}"
+									   CornerRadius="{TemplateBinding CornerRadius}"
+									   Padding="{TemplateBinding Padding}"
+									   AutomationProperties.AccessibilityView="Raw"
+									   VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+									   HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}">
+								<um:Ripple.Content>
+									<Grid>
+										<Grid.ColumnDefinitions>
+											<ColumnDefinition Width="Auto" />
+											<ColumnDefinition Width="*" />
+										</Grid.ColumnDefinitions>
+
+										<ContentPresenter x:Name="IconPresenter"
+														  Content="{Binding Path=(um:ControlExtensions.Icon), RelativeSource={RelativeSource TemplatedParent}}"
+														  MinWidth="18"
+														  HorizontalAlignment="Stretch"
+														  VerticalAlignment="Stretch"
+														  Visibility="{Binding Path=(um:ControlExtensions.Icon), RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource MaterialNullToCollapsedConverter}, FallbackValue=Collapsed, TargetNullValue=Collapsed}"
+														  VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+														  HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}" />
+
+										<ContentPresenter x:Name="ContentPresenter"
+														  Grid.Column="1"
+														  Margin="8,0"
+														  Content="{TemplateBinding Content}"
+														  ContentTemplate="{TemplateBinding ContentTemplate}"
+														  ContentTransitions="{TemplateBinding ContentTransitions}"
+														  FontFamily="{TemplateBinding FontFamily}"
+														  FontSize="{TemplateBinding FontSize}"
+														  FontWeight="{TemplateBinding FontWeight}"
+														  CharacterSpacing="{TemplateBinding CharacterSpacing}"
+														  HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+														  VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}" />
+									</Grid>
+								</um:Ripple.Content>
+							</um:Ripple>
+						</Grid>
+					</toolkit:ElevatedView>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+
+	<!-- OUTLINED-->
+	<Style x:Key="M3MaterialOutlinedButtonStyle"
+		   TargetType="Button">
+
+		<Setter Property="Foreground"
+				Value="{StaticResource PrimaryBrush}" />
+		<Setter Property="Background"
+				Value="Transparent" />
```

Comment:
I thought the Background would be SurfaceBrush

![image](https://user-images.githubusercontent.com/4793020/163510121-3bfae394-f15b-408d-8f2e-66f087df96eb.png)